docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`2628` Users will now see the correct total asset value when visiting an asset's detail page for a second time.
```

Comment:
Can you move the bugs to the bottom of the release notes along with the other bugs? This is a convention I try to follow so that (1) they are grouped together (2) since most bugs normally are from bugfixes branch we have as little merge conflicts as possible.

Reply:
ok I will do before merging